src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.Caching.cs

```diff
@@ -33,8 +33,6 @@ public sealed partial class JsonSerializerOptions
         ///
         /// If the <see cref="JsonSerializerOptions"/> instance is locked for modification, the method will return a cached instance for the metadata.
         /// </remarks>
-        [RequiresUnreferencedCode("Getting a metadata for a type may require reflection which depends on unreferenced code.")]
```

Comment:
This was not included in the original proposal, but it turns out we can also remove this annotation as a result of this change. Which makes me happy :-)